cmd/manager/Dockerfile

```diff
@@ -10,3 +10,6 @@ COPY --from=build-env /go/src/github.com/kubeflow/katib/cmd/manager/vizier-manag
 COPY --from=build-env /go/src/github.com/kubeflow/katib/pkg/manager/visualise /
 ENTRYPOINT ["./vizier-manager"]
 CMD ["-w", "kubernetes"]
+RUN GRPC_HEALTH_PROBE_VERSION=v0.2.0 && \
```

Comment:
Please move up this line between `WORKDIR /app` and `COPY --from=build-env...`.
Then we can use a cache even when there are some changes in the manager.

Reply:
I'll push a fix, thanks.